pkg-config.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=@PKG_CONFIG_LIBDIR@
+includedir=@PKG_CONFIG_INCLUDEDIR@
+
+Name: libdumb
```

Comment:
I think the name should likely be `dumb` to match the filename of the pkg-config file no?

Reply:
Though reading the spec it doesn't seem to matter much: https://people.freedesktop.org/~dbn/pkg-config-guide.html

Reply:
It's a human-readable name so it could be anything: DUMB, libdumb, Dynamic Universal Music Bibliotheque... or whatever else.